content/en/docs/cloud-api-adaptor/azure/_index.md

```diff
@@ -222,27 +182,151 @@ export AZURE_SUBNET_ID=$(az network vnet subnet list \
+
+{{% tab header="Last Release" %}}
+
+```bash
+export CAA_VERSION="0.8.0"
+curl -LO "https://github.com/confidential-containers/cloud-api-adaptor/archive/refs/tags/v${CAA_VERSION}.tar.gz"
+tar -xvzf "v${CAA_VERSION}.tar.gz"
+cd "cloud-api-adaptor-${CAA_VERSION}"
+```
+
+{{% /tab %}}
+
+{{% tab header="Latest Build" %}}
+
+```bash
+export CAA_BRANCH="main"
+curl -LO "https://github.com/confidential-containers/cloud-api-adaptor/archive/refs/heads/${CAA_BRANCH}.tar.gz"
+tar -xvzf "${CAA_BRANCH}.tar.gz"
+cd "cloud-api-adaptor-${CAA_BRANCH}"
+```
+
+{{% /tab %}}
+
+{{% tab header="DIY" %}}
+This assumes that you already have the code ready to use. On your terminal change directory to the Cloud API Adaptor's code base.
+{{% /tab %}}
+
+{{< /tabpane >}}
+
 ### CAA pod VM image
 
+{{< tabpane text=true right=true persist=header >}}
+{{% tab header="**Versions**:" disabled=true /%}}
+
+{{% tab header="Last Release" %}}
+
 Export this environment variable to use for the peer pod VM:
 
 ```bash
 export AZURE_IMAGE_ID="/CommunityGalleries/cococommunity-42d8482d-92cd-415b-b332-7648bd978eff/Images/peerpod-podvm-ubuntu2204-cvm-snp/Versions/${CAA_VERSION}"
 ```
 
-> **Note**:
->
-> - If you want to use the nightly build of pod VM image then follow the instructions [here](azure-advanced#use-nightly-builds).
-> - If you want to build the pod VM image yourself then follow the instructions [here](azure-advanced#build-a-custom-pod-vm-image).
+{{% /tab %}}
+
+{{% tab header="Latest Build" %}}
+
+An automated job builds the pod VM image each night at 00:00 UTC. You can use that image by exporting the following environment variable:
+
+```bash
+SUCCESS_TIME=$(curl -s \
+  -H "Accept: application/vnd.github+json" \
+  "https://api.github.com/repos/confidential-containers/cloud-api-adaptor/actions/workflows/azure-podvm-image-nightly-build.yml/runs?status=success" \
+  | jq -r '.workflow_runs[0].updated_at')
+
+export AZURE_IMAGE_ID="/CommunityGalleries/cocopodvm-d0e4f35f-5530-4b9c-8596-112487cdea85/Images/podvm_image0/Versions/$(date -u -jf "%Y-%m-%dT%H:%M:%SZ" "$SUCCESS_TIME" "+%Y.%m.%d" 2>/dev/null || date -d "$SUCCESS_TIME" +%Y.%m.%d)"
+```
+
+Above image version is in the format `YYYY.MM.DD`, so to use the latest image should be today's date or yesterday's date.
+
+{{% /tab %}}
+
+{{% tab header="DIY" %}}
+
+If you have made changes to the CAA code that affects the pod VM image and you want to deploy those changes then follow [these instructions](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/azure/build-image.md) to build the pod VM image. Once image build is finished then export image id to the environment variable `AZURE_IMAGE_ID`.
+
+{{% /tab %}}
+
+{{< /tabpane >}}
 
 ### CAA container image
 
+{{< tabpane text=true right=true persist=header >}}
+{{% tab header="**Versions**:" disabled=true /%}}
+
+{{% tab header="Last Release" %}}
+
+Export the following environment variable to use the latest release image of CAA:
```

Comment:
wouldn't the latest CAA_TAG be `latest`? or the git sha from the main branch's HEAD?